src/features/api-keys/routes.ts

```diff
@@ -60,7 +60,13 @@ export const createProvisioningApiKey: RequestHandler = async (req, res) => {
 	}
 
 	try {
-		const apiKey = await $createProvisioningApiKey(req, appId);
+		const apiKey = await $createProvisioningApiKey(
+			req,
+			appId,
+			req.body.provisioningKeyName
```

Comment:
Can you make this use `req.body.name` here please so that it can be standardized with the other api creation endpoints